mrtt-ui/src/components/RestorationAimsForm/restorationAimsQuestionsAndAnswers.js

```diff
@@ -0,0 +1,67 @@
+export default {
+  1: {
+    question: '3 .1 What are the ecological aim(s) of the project activities at the site?',
+    answers: [
+      'Increase mangrove area',
+      'Improve mangrove condition/halt or reduce degradation',
+      'Increase mangrove species richness',
+      'Offset mangrove loss from another area',
+      'Habitat protection',
+      'Increase native fauna/wildlife',
+      'Increase native flora/vegetation (non-mangrove)',
+      'Reduce invasive species',
+      'Increase ecological resilience',
+      'Increase habitat connectivity',
+      'Restore hydrological connectivity',
+      'Improve sediment dynamics',
+      'Improve nutrient cycling',
+      'Increase carbon storage and sequestration',
+      'None',
+      'Unknown',
+      'Other'
```

Comment:
Also, I am ignoring the 'other' thing for now (where we put a input with it, and submit that value). Plan on working on that very soon 

https://github.com/globalmangrovewatch/gmw-users/issues/51

Reply:
check out line `61-67` in projectDetailsForm. I have a question that uses the `questionMapping.js` file I mentioned above to set up the dataStructure. I was imagining extracting this function into something more generic, that we could use in each form